lib/cmd/fh3/notifications/list.js

```diff
@@ -0,0 +1,50 @@
+/* globals i18n */
+var common = require('../../../common.js');
+var fhreq = require("../../../utils/request");
+var util = require('util');
+
+module.exports = {
+  'desc' : i18n._('List notifications of a RHMAP App'),
+  'examples' : [{
+    cmd : 'fhc notifications list --app=<app> --env=<environment> --audit=<true>',
+    desc : i18n._('List notifications of the <app>')}],
+  'demand' : ['app', 'env'],
+  'alias' : {
+    'app':'a',
+    'env':'e',
+    'audit':'au',
+    'json': 'j',
+    0 : 'app',
+    1 : 'env',
+    2 : 'audit',
+    3 : 'json'
+  },
+  'describe' : {
+    'app'   : i18n._("Unique 24 character GUID of your cloud application."),
+    'env'   : i18n._("Unique 24 character GUID of the environment where this cloud application is deployed."),
+    'audit' : i18n._("Flag to list the notifications"),
```

Comment:
Let's not move commands blindly and reasearch api with current api in NGUI! 
It can happen that common command will use some old api (v2) that may be no longer maintained later.